### PR TITLE
Revert "Make the naming of underlying builtin for FP + Vector match Integer. (#15430)"

### PR DESCRIPTION
**Pull request by slavapestov**
This reverts commit 8e11af4012636b4e3915884ab749415372494843. "Fixes" rdar://problem/38780477.

### CODE REVIEW

**Comment by slavapestov** [comment]
@swift-ci Please smoke test

**Comment by slavapestov** [comment]
@swift-ci Please smoke test

File: stdlib/public/SDK/simd/simd.swift.gyb [hunk 1]
```diff
@@ -68,8 +68,8 @@ public struct ${vectype} {
   public init() { self.init(0) }
 
   @_transparent
-  public init(_ _value: Builtin.${llvm_vectype}) {
```
Comment (DougGregor) [inline stdlib/public/SDK/simd/simd.swift.gyb]: I missed this before, but please don't add an unlabeled initializer if you can at all avoid it. Those things cause exponential behavior in the type checker all over the place.

**Comment by slavapestov** [comment]
@swift-ci Please smoke test

**Comment by slavapestov** [comment]
@rudkx Fixed this properly in https://github.com/apple/swift/pull/15448 so I'm closing this PR.

File: stdlib/public/SDK/simd/simd.swift.gyb (hunk 1 above)
Comment (stephentyrone) [reply stdlib/public/SDK/simd/simd.swift.gyb]: @DougGregor Mark already fixed the immediate issue, but if we want to remove *all* the unlabeled inits from Builtins (Int, Float, simd), I'm happy to make a patch that does that. I just want them to be uniform.

**Pull request closed without merging**